docs/reference/NuGet-Client-SDK.md

```diff
@@ -9,14 +9,128 @@ ms.topic: conceptual
 
 # NuGet Client SDK
 
-The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
+The *NuGet Client SDK* refers to a group of NuGet packages centered around the [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol) and the [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging) packages from the [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client) GitHub repository.
 
 > [!Note]
 >  For documentation on the NuGet server protocol, please refer to the [NuGet Server API](~/api/overview.md).
 
-## Source code
+## Getting started
 
-The source code is published on GitHub in the project [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client).
+### Install the package
+
+```
+dotnet add NuGet.Protocol
+```
+
+## Examples
+
+You can find these examples on the [NuGet.Protocol.Samples](TODO) project on GitHub.
```

Comment:
⚠ TODO: Merge https://github.com/NuGet/Samples/pull/38 and update this link!

Reply:
Personally I think we should avoid duplication and ideally just link to the samples repo. 

My brain is basically going into, what if the APIs change here and we forget to update them both. 

Am I being too paranoid?

Reply:
> My brain is basically going into, what if the APIs change here and we forget to update them both.
> 
> Am I being too paranoid?

This is a real concern, but avoiding duplication between docs and the Samples repository isn't the solution. The documentation is the latest "source of truth" on how to use `NuGet.Protocol`. The Samples repository is a "test bed" to check that the documentation is correct. Why is that? Well, the user experience is better on the docs website than on the Samples repository:

1. The documentation website has better SEO. Given time, customers will be able to use Google to discover this documentation.
1. There's noise on the Samples repository due to C# boilerplate (the `.csproj`, the `Main` method, etc...)
1. On the docs website, the samples are free of boilerplate. I can also press the `Copy` button and paste it to my project. It's a snappy experience.
1. Down the road, I'd love to add Try .NET integration so that you can get Intellisense in the browser (example: press `Enter focus mode` on [this doc](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world?tutorial-step=5)).
1. We have better tools to keep the samples approachable as we add more examples on the documentation website. We can add Table of Contents, colored sections, links, etc... This is harder to do on the Samples repository

Reply:
It looks like we can link code snippets from a separate repository (see [this](https://docs.microsoft.com/en-us/contribute/code-in-docs#out-of-repo-snippet-references)). I'll play around with this feature

Reply:
Ok, the documentation now grabs the source code from the samples repository. Please let me know what you think! 😊